core/types/messages/state_root_update.go

```diff
@@ -0,0 +1,85 @@
+package messages
+
+import (
+	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+
+	"github.com/NethermindEth/near-sffl/aggregator/database/models"
```

Comment:
I agree with the dependency concern. The problem I have with it is for bindings, files are auto-generated in specific packages, and making a similar `To` implementation would be a bit problematic. That said, it's weird that models and bindings are treated differently then

Reply:
Not sure I understand what you mean.

> making a similar To implementation would be a bit problematic

Why?

> That said, it's weird that models and bindings are treated differently then

In what sense?

Reply:
Just for sake of clarity.
I meant: keep this file but move `NewStateRootUpdateMessageFromModel` and `ToModel` into `models`.
i.e
`message.NewStateRootUpdateMessageFromModel` -> `models.StateRootUpdateMessage::ToMessage`
`message.`

`message.StateRootUpdateMessage::ToModel` -> `models. message.NewStateRootUpdateMessageFromMessage`

Reply:
Because, as mentioned, the bindings files are auto-generated. They also need to have their own packages bc of clashes. It would be basically mixing auto-generated with custom files, doesn't seem good.
Again, don't see a problem doing this for models, I'm just trying to find a good way to treat them similarly.

Reply:
Let's discuss this after the meeting. Maybe I'm missing something

Reply:
We might want to revert this if we move DB models to core as well, but done in [2219ef9](https://github.com/NethermindEth/near-sffl/pull/49/commits/2219ef9255c715c288db6c6fa54b445b7bc6f9e4).